### ISSUE TITLE
Decode Bool

### ISSUE THREAD

**Issue by a72** [yaslab/CSV.swift#76]
You can add "0" and "1" values to the Boolean decoder ?
"0" = false
"1" = true

**Comment by yaslab** [comment]
I added the `BoolDecodingStrategy` option to `CSVRowDecoder` on pull request #77. Please see below for usage.

```swift
    func testBoolDecodingStrategy_custom() {
        let csv = """
            falseValue,trueValue
            0,1
            """
        do {
            let reader = try CSVReader(string: csv, hasHeaderRow: true)
            reader.next()

            let decoder = CSVRowDecoder()

            // You can provide a closure to convert `String` to `Bool` with this option.
            decoder.boolDecodingStrategy = .custom({ $0 != "0" })

            let row = try decoder.decode(BoolDecodingStrategyExample.self, from: reader)
            XCTAssertEqual(row.falseValue, false)
            XCTAssertEqual(row.trueValue, true)
        } catch {
            XCTFail("\(error)")
        }
    }
```

**Comment by yaslab** [comment]
I released #77 as version 2.3.1!
